.github/workflows/ci.yml

```diff
@@ -109,6 +116,35 @@ jobs:
           path: ./coverage/
           name: coverage
 
+  ef-tests:
+    runs-on: ubuntu-latest-32-cores
+    needs: build
+    steps:
+      - name: Checkout ef-tests
+        uses: actions/checkout@v3
+        with:
+          repository: kkrt-labs/ef-tests
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: "true"
+      - name: setup
+        run: make setup && make pull-kakarot
+      - uses: taiki-e/install-action@nextest
+
+      - name: make build directory
+        run: cd ./lib/kakarot && mkdir build
```

Comment:
do we really need this? maybe the actions/download-artifacts does it if the path doesn't exist

Reply:
I can try! 

Lemme push somthing and see if it can work without it.

Reply:
Seems to be working, removed and resolved.